ui/mirage/handlers/clients.js

```diff
@@ -29,13 +30,16 @@ function getSum(array, key) {
 }
 
 function getTotalCounts(array) {
+  const counts = CLIENT_TYPES.reduce((obj, key) => {
```

Comment:
Although the previous code was easier to read, this refactor is to minimize where we have to add new client types to make that process easier as client types grow

Reply:
nice! if i'm reading this file and understanding this comment correctly, would this mean that adding a new client type to Mirage would mean simply adding it to the `CLIENT_TYPES` const? if so, it'd be nice to document/leave a comment for that somewhere. 😀

Reply:
Almost, the client type also has to be added below (I commented there, too) But the const `CLIENT_TYPES` at least takes care of updating all of the utils. 

I can definitely add a comment! Wanted to make sure pulling the types into a const seemed reasonable to folks :)